src/attr/_compat.py

```diff
@@ -75,6 +79,11 @@ def metadata_proxy(d):
         return res
 
 else:
+    # Disabling coverage because we run the test suite under coverage only for
+    # the latest Python 3.
+    if sys.version_info[1] >= 6:  # pragma: nocover
```

Comment:
Should this be `or PYPY`, or do you not want to rely on that?

Reply:
Excellent point!